extensions/Push/lib/Connector/Phabricator.pm

```diff
@@ -55,12 +54,40 @@ sub send {
     my $logger = Bugzilla->push_ext->logger;
 
     my $data = $message->payload_decoded;
-    my $bug_data = $self->_get_bug_data($data) || return 0;
-    my $bug = Bugzilla::Bug->new( { id => $bug_data->{id}, cache => 1 } );
+    my $bug = $self->_get_bug_by_data($data) || return PUSH_RESULT_OK;
+
+    # Q: Should we check to see if a group we don't care about changed?
+    # If so, then this doesn't automatically mean we need to make req to Phab?
+    # ======================================================
+    $is_public = is_public($bug);
+
+    # Check to see if any synced groups have changed
+    # =================================================
+    my $phab_sync_groups = Bugzilla->params->{phabricator_sync_groups}
+        || ThrowUserError('invalid_phabricator_sync_groups');
```

Comment:
Change to: 
ThrowUserError('invalid_phabricator_sync_groups') unless $phab_sync_groups;